.github/workflows/push-translations.yml

```diff
@@ -0,0 +1,46 @@
+name: Push Translations to Mojito
+
+on:
+  push:
+    paths:
+      - client-participation/js/strings/en.js
+    branches:
+      - 320-continuous-localization
+
+jobs:
+  push-translations:
+    runs-on: ubuntu-latest
+    env:
+      MOJITO_VERSION: "0.110"
+      MOJITO_REPO: polis
+      MOJITO_CLI_SERVER: polis-translations.herokuapp.com
+      MOJITO_CLI_USERNAME: admin
+      MOJITO_CLI_PASSWORD: ${{ secrets.MOJITO_CLI_PASSWORD }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.0
+
+      - name: Cache Mojito CLI
+        uses: actions/cache@v2.0.0
+        with:
+          path: moojito-cli.jar
+          key: ${{ env.MOJITO_VERSION }}-cli
+
+      - name: Download Mojito CLI
+        run: |
+          wget --continue --progress=dot:giga --output-document mojito-cli.jar https://github.com/box/mojito/releases/download/v${MOJITO_VERSION}/mojito-cli-${MOJITO_VERSION}.jar
+
+      - name: Write Mojito CLI config
+        run: |
+          cat <<-EOF > application.properties
+            l10n.resttemplate.host=${MOJITO_CLI_SERVER}
+            l10n.resttemplate.port=443
+            l10n.resttemplate.scheme=https
+            l10n.resttemplate.authentication.credentialProvider=CONFIG
+            l10n.resttemplate.authentication.username=${MOJITO_CLI_USERNAME}
+            l10n.resttemplate.authentication.password=${MOJITO_CLI_PASSWORD}
+          EOF
```

Comment:
We could instead just copy this from a file at `.github/support/mojito-cli/application.properties` (envvar substitute can happen at runtime by the CLI itself, so it's a simple copy)

Reply:
Pushed this change.